applications/crossbar/priv/couchdb/schemas/system_config.pusher.json

```diff
@@ -12,10 +12,19 @@
             "default": {},
             "description": "pusher apple",
             "properties": {
+                "apns_topic": {
+                    "description": "APNs topic for push notifications payload",
+                    "type": "string"
+                },
                 "certificate": {
                     "description": "PEM-encoded key and certificate",
                     "type": "string"
                 },
+                "extra_headers": {
+                    "default": {},
+                    "description": "Additional headers for INVITEs sent to push devices in the UA",
+                    "type": "object"
```

Comment:
@danielfinke maybe we can describe here the headers that define kamailio behaviour. thoughts ? need help ?

Reply:
@lazedo I can document them here if you think that makes sense, but is there any other place where I should document supported headers on endpoints or originate reqs or anything else

Reply:
@danielfinke i was thinking on adding them as properties and allow additionalProperties (expand the definition of extra_headers